### PR TITLE
Azure gov and armchecker updates

### PR DESCRIPTION
**Pull request by Akreddy123**

### CODE REVIEW

File: 04-Continous-Assurance/Readme.md [hunk 1]
```diff
@@ -88,6 +88,10 @@ for your application.)
 	        [-AzureADAppName <AzureADAppName>]
 ```
 
+Note:
+
+For Azure environments otherthan Azure Cloud, don't forget to provide AutomationAccountLocation as the default value won't work in those environments.
```
Comment (ganesh-msft) [inline 04-Continous-Assurance/Readme.md]:
```suggestion
For Azure environments other than Azure Cloud, don't forget to provide AutomationAccountLocation as the default value won't work in those environments.
```

File: 07-Customizing-AzSK-for-your-Org/Readme.md [hunk 2]
```diff
@@ -145,6 +145,11 @@ Install-AzSKOrganizationPolicy -SubscriptionId <SubscriptionId> `
            -DepartmentName "IT" `
            -PolicyFolderPath "D:\ContosoPolicies"
 ```
+
+Note:
+
+For Azure environments otherthan Azure Cloud, don't forget to provide ResourceGroupLocation as the default value won't work in those environments.
```
Comment (ganesh-msft) [inline 07-Customizing-AzSK-for-your-Org/Readme.md]:
```suggestion
For Azure environments other than Azure Cloud, don't forget to provide ResourceGroupLocation as the default value won't work in those environments.
```

File: 04-Continous-Assurance/Readme.md (hunk 1 above)
Comment (Akreddy123) [reply 04-Continous-Assurance/Readme.md]: done

File: 07-Customizing-AzSK-for-your-Org/Readme.md (hunk 2 above)
Comment (Akreddy123) [reply 07-Customizing-AzSK-for-your-Org/Readme.md]: done

**Pull request merged**